src/library-authoring/author-library/LibraryAuthoringPage.jsx

```diff
@@ -268,11 +265,14 @@ const ButtonTogglesBase = ({
   library, setShowPreviews, showPreviews, sending, quickAddBehavior, intl,
 }) => (
   <>
-    <Button variant="success" className="mr-1" size="lg" disabled={sending} onClick={quickAddBehavior}>
+    {/* todo: either reimplement the scroll to the add components button functionality,
+              figure out a better UX for the add component button at the top, or just
+              remove it entirely */}
```

Comment:
Did you remove it as a UX consideration, or just to simplify the refactor temporarily?

Reply:
I'm fine with getting rid of the button, for now.  In the end it won't be a decision to make on our own, but we can always add the button back if somebody complains.

(What will eventually happen, I expect, is that this page will suffer a major UX overhaul.  Whoever develops the changes will likely get handed down a full design document.)